ktor-io/jvm/src/io/ktor/utils/io/ByteChannelSequentialJVM.kt

```diff
@@ -83,6 +83,25 @@ public class ByteChannelSequentialJVM(initial: IoBuffer, autoFlush: Boolean) : B
         return readAvailableSuspend(dst)
     }
 
+    override fun readAvailable(min: Int, block: (ByteBuffer) -> Unit): Int {
```

Comment:
why do we need `min` here?

Reply:
I added it to mirror API and behavior of `writeAvailable`. Nut I also think that it's useful and we can have it on its own.